### PR TITLE
Add Travis CI badge, and don't check external links for now

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Add link checker to site CI](https://trello.com/c/cAeprW7m/1625-add-link-checker-to-site-ci)

File: _bin/test [hunk 1]
```diff
@@ -23,4 +23,5 @@ echo "Testing: $target"
 
 docker run -ti \
     --net host \
-    soulshake/linkchecker --check-extern "$target"
+    soulshake/linkchecker "$target"
+    #soulshake/linkchecker --check-extern "$target"
```
Comment (ianmalott) [inline _bin/test]: Pretty sure line 26 was just changed in a recent PR, so are you intentionally reverting the line or should this diff not be in the commit?

Comment (soulshake) [inline _bin/test]: Yes I'm taking out `--check-extern` for now because it won't pass yet; I need to fix the grid.console.com redirect -_-

Comment (ianmalott) [inline _bin/test]: Okay, works for me. I think I'd vote for removing the commented line (it had me wondering if you had accidentally committed a debugging change), but not a big deal.

**Pull request merged**